source/_integrations/sensehat.markdown

```diff
@@ -140,8 +140,35 @@ group:
   kitchen:
     - group.sense_hat
 ```
-
-### Directions for installing on Raspberry Pi Raspbian Based installation:
+## Installation
+### On Home Assistant Operating System
+
+Things got complicated for poor _SenseHAT_ with Home Assistant Operating System. I found finally a way to use it.
+
+- Follow instructions [here](https://www.home-assistant.io/hassio/enable_i2c/) to enable i2c on Home Assistant Operating System
+- Install "Custom deps deployment" add-on. To do so
+    - add ``https://github.com/home-assistant/hassio-addons-development`` to your add-ons repositories
+    - select *Custom deps deployment* and enable it
+- Configure "Custom deps deployment" add-on
+    ```yaml
```

Comment:
Code fenced blocks need to be surrounded by an empty new line

Reply:
solved.